planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/test/src/utils.hpp

```diff
@@ -161,25 +161,26 @@ inline void generatePossibleCollisions(
   }
 }
 inline void addConstantVelocity(
-  autoware_planning_msgs::msg::PathWithLaneId & trajectory, double velocity)
+  autoware_auto_planning_msgs::msg::PathWithLaneId & trajectory, double velocity)
 {
   for (auto & p : trajectory.points) {
-    p.point.twist.linear.x = velocity;
+    p.point.longitudinal_velocity_mps = velocity;
   }
 }
-inline autoware_perception_msgs::msg::DynamicObject generateDynamicObject(double x)
+inline autoware_auto_perception_msgs::msg::PredictedObject generatePredictedObject(double x)
 {
-  autoware_perception_msgs::msg::DynamicObject obj;
+  autoware_auto_perception_msgs::msg::PredictedObject obj;
   obj.shape.dimensions.x = 5.0;
   obj.shape.dimensions.y = 2.0;
   tf2::Quaternion q;
-  obj.state.pose_covariance.pose.orientation = tf2::toMsg(q);
-  obj.state.orientation_reliable = true;
-  obj.state.twist_reliable = true;
-  obj.state.twist_covariance.twist.linear.x = 0;
-  obj.state.pose_covariance.pose.position.x = x;
-  obj.state.pose_covariance.pose.position.y = 0;
-  obj.semantic.type = autoware_perception_msgs::msg::Semantic::CAR;
+  obj.kinematics.initial_pose_with_covariance.pose.orientation = tf2::toMsg(q);
+  // TODO(murooka) twist/orientation reliable
+  // obj.kinematics.orientation_reliable = true;
+  // obj.kinematics.twist_reliable = true;
+  obj.kinematics.initial_twist_with_covariance.twist.linear.x = 0;
```

Comment:
How can I deal with reliable of predicted object kinematics since this message doesn't have.
then remove TODO(murooka)
@yukkysaito

Reply:
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/605/commits/efaafd118c4e5dc327724dcd6219c21b9d3017c0